docs/pages/blog/mui-x-v6-alpha-zero.md

```diff
@@ -0,0 +1,96 @@
+The next version's documentation is in the `next` subdomain.
+
+- [https://next.mui.com/x/react-data-grid/](https://next.mui.com/x/react-data-grid/)
+- [https://next.mui.com/x/react-date-pickers/](https://next.mui.com/x/react-date-pickers/)
+
+## What’s coming next?
+
+The following is a list of enhancements on the pipeline for the next few months. It contains the highlights that will make into the first versions.
+
+### Data Grid
+
+- **Extended customization abilities**.  
+   We want to empower more users to suit the data grid to complex use cases.
+
+  - [ApiRef in the community package](https://github.com/mui/mui-x/issues/6147).
+  - [Filtering on header](https://github.com/mui/mui-x/issues/6247).
+  - [Use the Data Grid internal components outside the grid](https://github.com/mui/mui-x/issues/2522).
+
+- **Improved Design**  
+   We’re polishing edges in terms of design and usability.
+
+  - [New column menu](https://github.com/mui/mui-x/issues/4929).
+  - [New column visibility panel](https://github.com/mui/mui-x/issues/5700).
+  - [Column Resize Bar only on hover](https://github.com/mui/mui-x/issues/1623).
+
+- **Rockstar feature**
+  - [Copy and paste from/to multiple cells](https://github.com/mui/mui-x/issues/199).
+
+### Date pickers
+
+- **Improved UX**  
+   On the previous version we focused a lot on developer experience and overall stability of the components. Now we’re pushing the lines to improve usability.
+  - [Remove the clock view on time pickers (for desktop)](https://github.com/mui/mui-x/issues/4483).
+  - [Visually edit a range by dragging date markers](https://github.com/mui/mui-x/issues/5311).
+  - [Range shortcuts](https://github.com/mui/mui-x/issues/4563).
+  - [A new text input for date and time values, retiring the mask solution (early preview)](https://next.mui.com/x/react-date-pickers/date-field/).
+
+:::info
+You can check our [roadmap](https://github.com/mui/mui-x/projects/1) for the full live list.
+:::
+
+## How to migrate?
+
+We’ve prepared a [migration guide](https://mui.com/x/react-data-grid/migration-v5/), and we’ll feed the page over time, as we make any breaking changes during the pre-releases. It lists every update you need to do on your code to use the most recent packages.
+
+We highly encourage you to try the new version. It is, after all, an improvement over the solid foundation we established with v5.
+
+## How to get involved?
+
+Please consider joining our alpha testers group channel and [giving us](https://forms.gle/vsBv6CLPz9h57xg8A) an user interview.  
+You'll get an insider's perspective of the development and can help us iterate early in the new features.
+
+As usual, we're glad to listen all the feedback, and you can also participate in the discussion by commenting on the features or reporting bugs in our [GitHub repository](https://github.com/mui/mui-x/issues/new/choose). Finally, you can follow every step of the development through our [changelog](https://github.com/mui/mui-x/releases).
```

Comment:
I guess we are missing the actual changelog of v6.0.0-alpha.0? Either as a reference or inclined.

Reply:
I added a link to the alpha zero release on first section of the blog.